test/mailers/previews/user_notifier_mailer_preview.rb

```diff
@@ -0,0 +1,8 @@
+# Preview all emails at http://localhost:3000/rails/mailers/user_notifier_mailer
+class UserNotifierMailerPreview < ActionMailer::Preview
```

Comment:
I didn't know that, cool!

btw, the indentation is still not correct, it should be two spaces instead of two tabs.